### PR TITLE
fix: validate input value and updates correctly adminSetup

### PR DESCRIPTION
**Pull request by Josmar-jr**
#### What problem is this solving?
Before, it not possible update the "expiration date" of quotes

#### How to test it?

1. Simulation Access the B2B Quotes settings page; 
2. Change the value of the "expiration period" field; 
3. Refresh the page; 

[Workspace](https://josmarjr--b2bstore005.myvtex.com/admin/b2b-quotes/settings)

#### Screenshots or example usage:
Screenshot 2024-09-12 at 13 51 45

https://github.com/user-attachments/assets/42358669-ed01-4f23-9968-8138523454f6

### CODE REVIEW

File: react/components/B2BQuotesLockingModal.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ const B2BQuotesLockingModal = () => {
           }
         >
           <div>
-            <h1>{formatMessage(messages.title)}</h1>
+            {/* <h1>{formatMessage(messages.title)}</h1> */}
```
Comment (ataideverton) [inline react/components/B2BQuotesLockingModal.tsx:162]: Please remove the comment

File: react/components/QuoteDetails/SaveButtons.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const SaveButtons = ({
               updatingQuoteState
             }
           >
-            <FormattedMessage id="store/b2b-quotes.create.button.save-for-later" />
+            fdsdf
```
Comment (ataideverton) [inline react/components/QuoteDetails/SaveButtons.tsx:33]: I thinks this is some test code that ended up on the PR

**Review by ataideverton — CHANGES_REQUESTED** [review]
Please add a changelog entry

File: react/components/QuoteDetails/SaveButtons.tsx (hunk 2 above)
Comment (Josmar-jr) [reply react/components/QuoteDetails/SaveButtons.tsx:33]: resolved

File: react/components/QuoteDetails/SaveButtons.tsx [hunk 3]
```diff
@@ -44,11 +44,7 @@ const SaveButtons = ({
               updatingQuoteState
             }
           >
-            {isSalesRep ? (
-              <FormattedMessage id="store/b2b-quotes.quote-details.save" />
-            ) : (
-              <FormattedMessage id="store/b2b-quotes.create.button.request-quote" />
-            )}
+            asdfasdf
```
Comment (ataideverton) [inline react/components/QuoteDetails/SaveButtons.tsx:47]: Outro pedaço de código de teste que subiu

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixed the change of expiration date inserted in the input
```
Comment (ataideverton) [inline CHANGELOG.md:12]:
```suggestion
- Fix validation and submit on lifespan settings for quotes
```

**Review by ataideverton — CHANGES_REQUESTED** [review]

File: react/components/QuoteDetails/SaveButtons.tsx (hunk 3 above)
Comment (Josmar-jr) [reply react/components/QuoteDetails/SaveButtons.tsx:47]: resolve

**Review by ataideverton — APPROVED** [review]

**Pull request merged**